projects/dotcms/src/lib/api/tests/DotApiForm.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { DotApiForm } from '../DotApiForm';
+import { DotCMSContentTypeField } from 'dotcms/lib/models';
+
+function validateForm(formScript: string): void {
```

Comment:
the whole string contains a lot of break lines and spaces that makes it hard to compare

Reply:
This `it` have 8 expect, is too much we had issues for track down broken test because of this in the pass.

In this case here is you can do:

`console.log(container.innerHTML)`

Copy what is there and then you can do:

```
expect(container.innerHTML).toBe(`
<h1>    Hello</h1>
    <label>stuff</label>
<input type="   check">
`)
```

And that way any string no matter spaces or break likes will be expect correctly in one expect... I think 😅

Reply:
you could remove them before compare